js/errors.js

```diff
@@ -26,11 +28,15 @@
      * @param  {string} subMessage  technical details about the error. Appear in collapsible span in the modal box
      * @constructor
      */
-    function ERMrestError(code, status, message, subMessage) {
+    function ERMrestError(code, status, message, subMessage, redirectPath) {
```

Comment:
Comment what this `redirectPath` is. And I don't think you need the `errorData` object. Just flatten the attribute (`this.redirectPath = redirectPath`)

Reply:
errorData is an object that is being used in [`Chaise`](https://github.com/informatics-isi-edu/chaise/blob/facetErrors/common/errors.js#L249:L261) to contain extra information regarding errors. I added here to align it with chaise and also make it more readable at chaise repo when extracting redirect link.